tests/test_formpack_internals.py

```diff
@@ -61,6 +61,46 @@ def test_to_xml_fails_when_null_labels():
     fp[0].to_xml()
 
 
+def test_groups_disabled():
+    scontent = {'content': {
+                   'survey': [
+                              {'type': 'text','name': 'n1', 'label': ['aa']},
+                              {'type': 'begin_group', 'name': 'nada'},
+                              {'type': 'note', 'name': 'n2', 'label': ['ab']},
+                              {'type': 'end_group'},
+                              ],
+                   'translations': ['en'],
+                   'translated': ['label'],
+                  }
+                }
+
+    (ga, gz) = [scontent['content']['survey'][nn] for nn in [1, 3]]
+    # verify that "ga" and "gz" variables point to group begin/end
+    assert ga['type'] == 'begin_group'
+    assert gz['type'] == 'end_group'
+    assert ga.get('disabled') == gz.get('disabled') == None
+
+    # verify values before setting "disabled=TRUE"
+    fp = FormPack(scontent, id_string='xx')
+    ss = [ss for ss in fp[0].sections.values()][0]
```

Comment:
is this like `list(fp[0].sections.values())[0]`? it causes me a bit of anguish that, under Python 3, we can't:
```
In [2]: odicty = OrderedDict(zip(list('abcdef'), list('123456')))

In [3]: odicty.values()[0]
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-5287d4b9ed63> in <module>()
----> 1 odicty.values()[0]
```
it is an _ordered_ dict after all :roll_eyes:

alas, the least-gross way i can think of to deal with this common pattern is:
```
In [4]: next(iter(odicty.values()))
Out[4]: '1'
```